core/frontend/src/components/vehiclesetup/overview/OnboardSensors.vue

```diff
@@ -0,0 +1,185 @@
+              <td>{{ baro.busType }} {{ baro.bus }}</td>
+              <td>{{ `0x${baro.address}` }}</td>
+              <td>{{ baro_status(baro) }}</td>
+            </tr>
+          </tbody>
+        </template>
+      </v-simple-table>
+    </v-card-text>
+  </v-card>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+
+import Notifier from '@/libs/notifier'
+import autopilot_data from '@/store/autopilot'
+import autopilot from '@/store/autopilot_manager'
+import mavlink from '@/store/mavlink'
+import { printParam } from '@/types/autopilot/parameter'
+import { parameters_service } from '@/types/frontend_services'
+import decode, { deviceId } from '@/utils/deviceid_decoder'
+import mavlink_store_get from '@/utils/mavlink'
+
+const notifier = new Notifier(parameters_service)
+
+export default Vue.extend({
+  name: 'OnboardSensors',
+  computed: {
+    imus() : deviceId[] {
+      const ins_params = autopilot_data.parameters.filter(
+        (parameter) => parameter.name.startsWith('INS_ACC') && parameter.name.endsWith('_ID') && parameter.value !== 0,
+      )
+      return ins_params.map((parameter) => decode(parameter.name, parameter.value)) as deviceId[]
+    },
+    compasses() {
+      const compass_params = autopilot_data.parameters.filter(
+        (param) => param.name.startsWith('COMPASS') && param.name.includes('DEV_ID') && param.value !== 0,
+      )
+      return compass_params.map((parameter) => decode(parameter.name, parameter.value))
+    },
+    baros() {
+      const baro_params = autopilot_data.parameters.filter(
+        (param) => param.name.startsWith('BARO') && param.name.includes('DEVID') && param.value !== 0,
+      )
+      return baro_params.map((parameter) => decode(parameter.name, parameter.value))
+    },
+  },
+  mounted() {
+    mavlink.setMessageRefreshRate({ messageName: 'SCALED_PRESSURE', refreshRate: 1 })
+    mavlink.setMessageRefreshRate({ messageName: 'SCALED_PRESSURE2', refreshRate: 1 })
+    mavlink.setMessageRefreshRate({ messageName: 'VFR_HUD', refreshRate: 1 })
+  },
+  methods: {
+    imu_is_calibrated(imu: deviceId) {
+      const param_radix = imu.param.split('_ID')[0]
+      const offset_params_names = [`${param_radix}OFFS_X`, `${param_radix}OFFS_Y`, `${param_radix}OFFS_Z`]
+      const scale_params_names = [`${param_radix}SCAL_X`, `${param_radix}SCAL_Y`, `${param_radix}SCAL_Z`]
+      const offset_params = offset_params_names.map(
+        (name) => autopilot_data.parameters.filter((parameter) => parameter.name === name).first(),
+      )
+      const scale_params = scale_params_names.map(
+        (name) => autopilot_data.parameters.filter((parameter) => parameter.name === name).first(),
+      )
+      const is_at_default_offsets = offset_params.map((param) => param?.value).every((value) => value === 0.0)
+      const is_at_default_scale = scale_params.map((param) => param?.value).every((value) => value === 1.0)
+      return !is_at_default_offsets || !is_at_default_scale
```

Comment:
We need to check if the list is not empty... because...

![image](https://user-images.githubusercontent.com/1215497/214405735-20cbbcbb-0534-48d0-a24e-c14d21b329d7.png)